internal/entities/properties/constants.go

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright 2024 Stacklok, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package properties
+
+// Repository property keys
+const (
+	RepoPropertyName   = "name"
+	RepoPropertyOwner  = "owner"
+	RepoPropertyRepoID = "repo_id"
+	// TODO: is this GH-specific? If yes, it should be moved under a provider-specific property
+	RepoPropertyDeployURL     = "deploy_url"
+	RepoPropertyCloneURL      = "clone_url"
+	RepoPropertyIsPrivate     = "is_private"
+	RepoPropertyIsArchived    = "is_archived"
+	RepoPropertyIsFork        = "is_fork"
+	RepoPropertyDefaultBranch = "default_branch"
+	RepoPropertyLanguages     = "properties.languages"
```

Comment:
I'm unsure about `properties.languages`. Yes, we need languages, I just wanna figure out what's a good way to express this property.

Reply:
I removed them for now. I think we need to figure out a better way to express the nested properties anyway, this was too much of a hack